lib/generators/has_friendship/has_friendship_generator.rb

```diff
@@ -17,10 +17,21 @@ def self.next_migration_number(path)
 
   def create_migration_file
     migration_template 'create_friendships.rb',
-                       'db/migrate/create_friendships.rb'
+                       'db/migrate/create_friendships.rb', migration_version: migration_version
```

Comment:
Is this something required from Rails 5 and onwards? Could you link me to a source?